src/Interfaces/AbstractPolyhedron_functions.jl

```diff
@@ -578,3 +579,30 @@ function chebyshev_center(P::AbstractPolyhedron{N};
     end
     return c
 end
+
+"""
+    an_element(P::AbstractPolyhedron{N}) where {N<:Real}
+
+Return some element of a convex set.
+
+### Input
+
+- `S` -- convex set
+
+### Output
+
+An element of a polyhedron.
+
+### Algorithm
+
+An element of the polyhedron is obtained by evaluating its support vector along
+direction ``[1, 0, …, 0]``.
+"""
+function an_element(P::AbstractPolyhedron{N}) where {N<:Real}
+    if dim(P) == -1
```

Comment:
Maybe store `dim(P)`?

Reply:
ok

Reply:
(i think there is no difference in performance, just for readability)